ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/ldes/eventstream/services/EventStreamConverterImpl.java

```diff
@@ -55,9 +52,17 @@ private List<Statement> addViewStatements(List<TreeNode> views) {
 
 		views.forEach(view -> {
 			Resource viewResource = createResource(view.getFragmentId());
-
 			statements.add(createStatement(viewResource, RDF_SYNTAX_TYPE, createResource(TREE_NODE_RESOURCE)));
-			statements.addAll(relationStatementConverter.getRelationStatements(view.getRelations(), viewResource));
+			view.getRelations()
+					.forEach(treeRelation -> {
+						TreeRelationResponse treeRelationResponse = new TreeRelationResponse(treeRelation.treePath(),
```

Comment:
Maybe we can add a static constructor that constructs an instance from a TreeRelation? Then we don't have to copy paste the entire object here and in the other converterImpl.

Reply:
There's actually an explicit reason why I don't want to include TreeRelation into TreeRelationResponse.
In my view:
* TreeRelation is a value object in the current domain (one bounded context)
* TreeRelationResponse is a value object in fetching (a bounded context I'm trying to isolate)
* Between those I would like to implement an anti-corruption layer so that changes in TreeRelation do not automatically flow to TreeRelationResponse. I'll make this more clear in the follow-up PRs.
See this image from the blue book.
![image](https://user-images.githubusercontent.com/104509963/219324669-dafc126d-7020-443d-932d-aa937b8a6d6f.png)